AISKU/src/AISku.ts

```diff
@@ -259,8 +261,40 @@ export class AppInsightsSku implements IApplicationInsights {
                     _self.emptyQueue();
                     _self.pollInternalLogs();
                     _self.addHousekeepingBeforeUnload(_self);
+
+                    _addUnloadHook(onConfigChange(cfgHandler, () => {
+                        if (!_throttleMgr){
+                            _throttleMgr = new ThrottleMgr(_core);
+                        }
+                        //  && !_config.throttleMgrCfg.disabled
+                        if (_throttleMgr && !_throttleMgr.isReady() && _config.extensionConfig && _config.extensionConfig[_cfgSyncPlugin.identifier]) {
+                            // set ready state to true will automaatically trigger flush()
+                            _throttleMgr.onReadyState(true);
+                        }
+
+                        if (!_iKeySentMessage){ //should not send again
+                            if (!_config.connectionString && !isFeatureEnabled("disableIkeyDeprecationMessage", _config)) {
+                                _throttleMgr.sendMessage( _eInternalMessageId.InstrumentationKeyDeprecation, "Instrumentation key support will end soon, see aka.ms/IkeyMigrate");
```

Comment:
As the `aka.ms/xxxx` are Microsoft Global, we should make these more unique to us, a bit like the `JsActiveCdn`. Maybe that should be even more unique... like
`JsSdkActiveCdn`, `JsSdkIKeyMigrate`, `JsSdkLdrVer` etc
As other people / teams in the company may also want to call things "Snippet" and in the case of the `IkeyMigrate` this is an Azure Monitor team wide change so it actually affects all SDK's and all users. So claiming might get tricky when other teams also want to do this, by keeping more specific to our team we have a greater level of justification for note being kicked out of our usage 😄

Reply:
Actually the message on this one I think is consistent with the general support message, we could shorten slightly dropping the "will end soon"

"Instrumentation key support, see aka.ms/....." or maybe
"See Instrumentation key support at aka.ms/....."